composer.json

```diff
@@ -58,7 +58,8 @@
         "phpstan/phpstan-phpunit": "^1.3.11",
         "phpstan/phpstan-strict-rules": "^1.5.1",
         "squizlabs/php_codesniffer": "^3.7.2",
-        "symfony/filesystem": "^6.2.10"
+        "symfony/filesystem": "^6.2.10",
+        "vimeo/psalm": "^5.11"
```

Comment:
I've removed `psalm` fro ParaTest 7 because of a few incompatibilites with `PHPStan` with zero gains: is there a valid reason to re-introduce it? If not I'd prefer to not have it

Reply:
Psalm is quite a lot faster than phpstan, and it offers many additional features like sealed shapes, list types, and even security analysis (though not really applicable in this project).

I've mainly added it because it's quite a lot faster than phpstan, and during iteration I found it easier to just run psalm instead of phpstan :)